app/ApiClients/GraylogApi.php

```diff
@@ -145,7 +145,11 @@ public function getAddresses(Device $device)
                 ->merge($device->ipv6->pluck('ipv6_address'));
         }
 
-        return $addresses->filter()->unique();
+        return $addresses->filter(
```

Comment:
You should only filter 127.0.0.1 from the addresses tables, not when they are hostname or ip

Reply:
> You should only filter 127.0.0.1 from the addresses tables, not when they are hostname or ip. So basically, move this filter up a few lines.

Changed IPv6 to uncompressed form and moved the filter up to line 143.

>  Also, try not to use double negatives it makes things hard to understand: return $address == '127.0.0.1' || $address == '0000:0000:0000:0000:0000:0000:0000:0001';

Not needed anymore but this does not the same as `return $address != '127.0.0.1' && $address != '::1';` or did I miss something? Your code will return only `127.0.0.1` and `::1`, my code will return everything **except** `127.0.0.1` and `::1`, or am I wrong?